<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>100</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>卖家：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_itemDetail">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>15</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>商品详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>底价：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agree">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>55</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_itemName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_seller">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>卖家：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_startPrice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>底价：</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>98</y>
     <width>521</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>521</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>15</y>
     <width>3</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_disagree">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>55</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>不通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chatSeller">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>15</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>联系卖家</string>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>btn_itemDetail</zorder>
  <zorder>label_3</zorder>
  <zorder>btn_agree</zorder>
  <zorder>lbl_itemName</zorder>
  <zorder>lbl_seller</zorder>
  <zorder>lbl_startPrice</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line</zorder>
  <zorder>btn_disagree</zorder>
  <zorder>btn_chatSeller</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
